packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1614,8 +1614,9 @@ export const InvalidContentEntryDataError = {
 	title: 'Content entry data does not match schema.',
 	message(collection: string, entryId: string, error: ZodError) {
 		return [
-			`**${String(collection)} → ${String(entryId)}** data does not match collection schema.`,
-			...error.errors.map((zodError) => zodError.message),
+			`**${String(collection)} → ${String(entryId)}** data does not match collection schema.\n`,
+			...error.errors.map((zodError) => `  **${zodError.path.join('.')}**: ${zodError.message}`),
```

Comment:
When adding the new error I noticed that these were missing the path, which could cause unhelpful errors

Reply:
2 things:
1. Why do you need `String(...)` when the type is already `string`?
2. I think the formatting of the error should happen where the throw happens rather than inside the error data, see https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/README.md#shape

Reply:
Can you provide two screenshots: one before and after the change?

Reply:
This is a very old error - I just added the `zodError.path.join('.')` bit. I followed the same pattern for the one I added. You're right the string cast isn't needed, but I think that standardising the formatting of the zod error does make sense

Reply:
Before:
![image](https://github.com/user-attachments/assets/93866f2e-a204-4cf1-a9cf-e744f894fc6e)

After:
![image](https://github.com/user-attachments/assets/3969ce5e-4c38-4396-9217-d5ad97c2cbe8)

See the field name `title` has been added

Reply:
Ah sorry I didn't notice it was an old one